eisenhower-matrix/src/main/webapp/script.js

```diff
@@ -1,46 +1,63 @@
+// check login status on load
+document.addEventListener("DOMContentLoaded", function () {
+  // every time there is a submission, fetch tasks
+  getTasks();
+<<<<<<< HEAD
```

Comment:
Can we get rid of this marker?

Reply:
oops! yes

Reply:
Sorry one last thing, the api key changed so can one of us update that real quick?